content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -32,19 +32,36 @@ mitigate some future vulnerabilities too.
 {{% thirdparty-content single="true" %}}
 <!-- if adding another runtime in the future, omit the single setting -->
 
-This is a Linux only feature. In addition, support is needed in the 
+This is a Linux-only feature and support is needed in Linux for idmap mounts.
+This means:
+
+* The filesystem you use for `/var/lib/kubelet/pods/`, or the custom directory
+  you configure for this, needs idmap mount support
+* All the filesystems used in the pod need support for idmap mounts.
+
+In practice this means you need at least Linux 6.3, as tmpfs started supporting
+idmap mounts in that version. This is usually needed as several Kubernetes
+features use tmpfs (the service account token that is mounted by default uses a
```

Comment:
```suggestion
features use tmpfs (the ServiceAccount tokens that are mounted by default use a
```

Reply:
Hmm, are you sure? There is only one `token` file in the service account that is mounted:

```
# ls /run/secrets/kubernetes.io/serviceaccount/
ca.crt	namespace  token
```

And the token file seems to contain one token

Reply:
I was trying to talk about the generals. In implementation, we are only mounting one ServiceAccount token today, even that token can be ignored.